Casks/nordic-nrf-command-line-tools.rb

```diff
@@ -2,7 +2,7 @@
   version "10.18.1"
   sha256 "e060ddf2044677f08c4f825ee31dfc8525c26f557abcb700758e1193bc556c9f"
 
-  url "https://www.nordicsemi.com/-/media/Software-and-other-downloads/desktop-software/nrf-command-line-tools/sw/Versions-#{version.major}-x-x/#{version.dots_to_hyphens}/nrf-command-line-tools-#{version}-darwin.dmg"
+  url "https://nsscprodmedia.blob.core.windows.net/prod/software-and-other-downloads/desktop-software/nrf-command-line-tools/sw/versions-#{version.major}-x-x/#{version.dots_to_hyphens}/nrf-command-line-tools-#{version}-darwin.dmg", verified: "nsscprodmedia.blob.core.windows.net/"
```

Comment:
```suggestion
  url "https://nsscprodmedia.blob.core.windows.net/prod/software-and-other-downloads/desktop-software/nrf-command-line-tools/sw/versions-#{version.major}-x-x/#{version.dots_to_hyphens}/nrf-command-line-tools-#{version}-darwin.dmg",
      verified: "nsscprodmedia.blob.core.windows.net/"
```
It's really easy to not even see the `verified` bit.

Reply:
#3180